client/src/modules/general-ledger/templates/action.cell.html

```diff
@@ -7,18 +7,18 @@
 
     <ul class="dropdown-menu-right" uib-dropdown-menu>
       <li>
-        <a ng-href="/reports/finance/account_report/?{{ grid.appScope.slip('pdf', grid.appScope.filtersSlip, row.entity.id) }}" download="{{GENERAL_LEDGER.ACCOUNT_SLIP | translate }}">
+        <a ng-href="/reports/finance/account_report/?{{ grid.appScope.slip('pdf', grid.appScope.filtersSlip, row.entity.id) }}" translate-attr="{ download: '{{GENERAL_LEDGER.ACCOUNT_SLIP}}' }">
```

Comment:
This can just be `{ download : 'GENERAL_LEDGE.ACCOUNT_SLIP' }`.  There is no need to use the extra `{{ }}`s.